app/containers/message/Reply.js

```diff
@@ -156,9 +156,8 @@ const Reply = React.memo(({
 			if (!url.startsWith('http')) {
 				url = `${ baseUrl }${ url }`;
 			}
-			url = `${ url }?rc_uid=${ user.id }&rc_token=${ user.token }`;
 		}
-		openLink(url, theme);
+		Navigation.navigate('GenericWebView', { uri: url, headers: { 'x-user-id': user.id, 'x-auth-token': user.token }, title: attachment.title });
```

Comment:
This is wrong.
You're forcing the link to be opened from the GenericWebView without knowing what it is.
You can do it inside the `file` if condition, but leave the `openLink` the way it was.
Tap on Reply from `#react-native-notifications` channel and you'll see it opening GenericWebView instead of open links using default behavior.